py/frequenz/dispatch/v1/__init__.py

```diff
@@ -1,4 +1,4 @@
-"""Frequenz Dispatch API.
+# License: MIT
+# Copyright © 2022 Frequenz Energy-as-a-Service GmbH
```

Comment:
2023

Reply:
This is only adding the header, but the file was published in 2022.